classes/class3/hw3/routes/cat.js

```diff
@@ -0,0 +1,85 @@
+var mongoose = require('mongoose');
+
+var catName = ['Gizmo', 'Beauregard', 'Baby Girl', 'Han Solo', 'Trisha', 'Quat', 'Arbitration', 'Darth', 'Saint Ashlar', 'Willa Catter',
+	'Curtains', 'Seurat', 'Fox', 'Possum' ,'Akeisha','Manuel','Nika','Rossini','Teddy Bear','Katman','Pooker','Jack Sparrow','Gecko','Tanier','Berkeley'];
+var catColors = ['brown', 'red', 'black', 'white', 'grey', 'orange', 'yellow'];
+
+var Cat = mongoose.model('Cat',{
+	age: Number,
+	name: String,
+	color: [String]
+});
+
+var cats = {};
+
+cats.newCat = function(req, res){
+	var colorList = [];
+	for (var i = 0; i < Math.floor(Math.random()*3)+1; i++) {
```

Comment:
Improperly indented code is not valid code. In the future we will not accept code that is this improperly indented, since it is way to hard for us to read.